src/views/open-api-docs-view/components/dev-code-block/index.tsx

```diff
@@ -0,0 +1,30 @@
+import { CSSProperties, ReactNode } from 'react'
+import classNames from 'classnames'
+import s from './dev-code-block.module.css'
+
+/**
+ * A temporary component for prototyping and scaffolding.
+ *
+ * Context: sometimes it feels useful to have some visual thing on the page that
+ * shows what props & data I have access to in a component when prototyping.
+ */
+export function DevCodeBlock({
```

Comment:
A bit of a weird component, i've found this type of thing helpful in prototyping and early implementation stages to show the available data flowing into stubbed-in sections of the page. Idea would of course be to remove any uses of `dev-code-block` as we complete the implementation of each section.